docs/admin/access.rst

```diff
@@ -63,20 +65,18 @@ project.
 
 .. image:: /screenshots/project-access.webp
 
-The default value can be changed by :setting:`DEFAULT_ACCESS_CONTROL`.
+Access mode can also be changed by setting :setting:`DEFAULT_ACCESS_CONTROL`.
 
 .. note::
 
-    Even for `Private` projects, some info about your project will be exposed:
-    statistics and language summary for the whole instance will include counts
-    for all projects despite the access control setting.
-    Your project name and other information can’t be revealed through this.
+    `Private` projects still expose counts for all projects in their
+    respective statistics and language summary.
+    This does not reveal project name or other info.
```

Comment:
This makes it IMHO less clear. The intention is to make clear that even private projects are reflected in site-wide statistics, such as global or language ones.